README.rst

```diff
@@ -209,6 +209,45 @@ A Personal Access Token is scoped to a Seam Console user. Obtain one from the Se
   # List workspaces authorized for this Personal Access Token
   workspaces = seam.workspaces.list()
 
+Webhooks
+~~~~~~~~
+
+The Seam API implements webhooks using `Svix <https://www.svix.com>`_. This SDK exports a thin wrapper ``SeamWebhook`` around the svix package. Use it to parse and validate Seam webhook events.
+
+Refer to the `Svix docs on Consuming Webhooks <https://docs.svix.com/receiving/introduction>`_ for an in-depth guide on best-practices for handling webhooks in your application.
+
+.. code-block:: python
```

Comment:
Not sure if you did this, but let's run it through https://black.vercel.app/ to check syntax